.clj-kondo/methodical/methodical/hooks/methodical/macros.clj

```diff
@@ -114,21 +114,24 @@
                         [(hooks/token-node 'do)
                          multimethod]
                         (filter some? (vals other-stuff))
-                        [(hooks/list-node
-                          (list*
-                           (hooks/token-node 'fn)
-                           (hooks/token-node (if (contains? #{nil :around} (some-> (:qualifier parsed) hooks/sexpr))
-                                               'next-method
-                                               '__FN__NAME__THAT__YOU__CANNOT__REFER__TO__))
-                           fn-tail))]))]
+                        [(-> (hooks/list-node
+                              (list*
+                               (hooks/token-node 'fn)
+                               (hooks/token-node (if (contains? #{nil :around} (some-> (:qualifier parsed) hooks/sexpr))
+                                                   'next-method
+                                                   '__FN__NAME__THAT__YOU__CANNOT__REFER__TO__))
+                               fn-tail))
+                             (vary-meta update :clj-kondo/ignore conj :redundant-fn-wrapper))]))]
```

Comment:
@markbastian I saw you asking how to do a `#_:clj-kondo/ignore` from within a hook (I had the same question) and apparently this is how.

Reply:
Mentioned elsewhere but this only seems to work for certain linters.